engines/config-query-sparql/config/config-default-v4-2-0.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
I guess you copied this from `config-default.json`.
Could you start from `config-default-v4-1-0.json` instead?